api/src/main/java/org/openmrs/module/fhir2/providers/r4/EncounterFhirResourceProvider.java

```diff
@@ -196,16 +188,17 @@ public IBundleProvider getEncounterEverything(@IdParam IdType encounterId) {
 	 * @param date restrict by encounter date
 	 * @param status if set to active, when determined encounters to include, exclude encounters that
 	 *            *only* have completed, cancelled or declined medication requests
-	 * @param patientSearchTerm restrict to encounters for patients who name or identifier matches the
+	 * @param patientSearchTerm restrict to encounters for patients whose name or identifier matches the
 	 *            search term
-	 * @param location restrict to encounters at a certain location
+	 * @param location restrict to encounters at certain location(s)
+	 * @param type restrict to encounters of certain (encounter) type(s)
```

Comment:
And the (s) would seem to indicate that multiple locations or encounter types is possible?